lib/phoenix/live_dashboard/helpers/helpers.ex

```diff
@@ -223,9 +223,9 @@ defmodule Phoenix.LiveDashboard.LiveHelpers do
   """
   def live_info(_socket, %{info: nil}), do: nil
 
-  def live_info(socket, %{info: {title, params}, node: node, page: page}) do
+  def live_info(socket, %{info: {title, params}, node: node} = page) do
```

Comment:
This function and `live_modal` should be in `PageLive`, what do you think?

Reply:
Sounds good to me. I think we can move everything that is not reused to page_live itself. :)